code/modules/materials/Mat_Processing.dm

```diff
@@ -266,6 +266,15 @@ TYPEINFO(/obj/item/device/matanalyzer)
 					for(var/datum/material_property/X in W.material.getMaterialProperties())
 						var/value = W.material.getProperty(X.id)
 						boutput(user, SPAN_NOTICE("• [X.getAdjective(W.material)] ([value])"))
+					switch(W.material.getAlpha())
+						if(-INFINITY to 80)
+							boutput(user, SPAN_NOTICE("• extremely transparent ([W.material.getAlpha()])"))
+						if(80 to 130)
+							boutput(user, SPAN_NOTICE("• very transparent ([W.material.getAlpha()])"))
+						if(130 to 180)
+							boutput(user, SPAN_NOTICE("• transparent ([W.material.getAlpha()])"))
+						if(180 to INFINITY)
+							boutput(user, SPAN_NOTICE("• opaque ([W.material.getAlpha()])"))
```

Comment:
Might read better if it went "opaque, slightly transparent, transparent, very transparent"